<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_bt_dish_table_tableset2_set</class>
 <widget class="QDialog" name="w_bt_dish_table_tableset2_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
       <item row="1" column="1">
        <widget class="lds_doublespinbox" name="doubleSpinBox_int_unit_minute">
         <property name="suffix">
          <string>分钟</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="lds_doublespinbox" name="doubleSpinBox_num_unit_amount"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>服务率</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="lds_ComboBox" name="comboBox_ch_service_mode"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>定额服务费</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="lds_doublespinbox" name="doubleSpinBox_num_lowcost"/>
       </item>
       <item row="5" column="0">
        <widget class="lds_ComboBox" name="comboBox_ch_low_mode"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>收费单位</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>单位时间收费额</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="lds_doublespinbox" name="doubleSpinBox_int_rate">
         <property name="suffix">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="lds_doublespinbox" name="doubleSpinBox_num_ration"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="lds_key3x4" name="widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_exit">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>lds_doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_key3x4</class>
   <extends>QWidget</extends>
   <header>lds_key3x4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
